### ISSUE TITLE
State value has wrong type (js-controller 3.3.x)

### ISSUE THREAD

**Issue by Feuer-sturm** [DrozmotiX/ioBroker.coronavirus-statistics#237]
js-controller: 3.3.1
Admin 5.0.11
COVID-19 live information Adapter: 0.7.0-4

Using new js-controller 3.3.1 will throw the following warnings:

This is just an extract from the log file. There are more states which are affected
```
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.country_Top_5.5.criticalPerOneMillion" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.country_Top_5.5.oneCasePerPeople" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.country_Top_5.5.oneDeathPerPeople" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.country_Top_5.5.oneTestPerPeople" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.country_Top_5.5.population" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.country_Top_5.5.recoveredPerOneMillion" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.country_Top_5.5.tests" has wrong type "number" but has to be "string"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.country_Top_5.5.testsPerOneMillion" has wrong type "number" but has to be "string"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.country_Top_5.5.todayRecovered" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.global_totals.activePerOneMillion" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.global_totals.criticalPerOneMillion" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.global_totals.oneCasePerPeople" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.global_totals.oneDeathPerPeople" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.global_totals.oneTestPerPeople" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.global_totals.population" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.global_totals.recoveredPerOneMillion" has wrong type "number" but has to be "value"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.global_totals.tests" has wrong type "number" but has to be "string"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.global_totals.testsPerOneMillion" has wrong type "number" but has to be "string"
coronavirus-statistics.0 (805) State value to set for "coronavirus-statistics.0.global_totals.todayRecovered" has wrong type "number" but has to be "value"
```

and this warning was thrown as well:
` coronavirus-statistics.0 (1033) State attribute definition missing for + undefined`

**Comment by mediaDS** [comment]
Same here, Log:

```
coronavirus-statistics.0	2021-05-10 16:55:37.086	info	(17706) State value to set for "coronavirus-statistics.0.Germany.Kreis.Stendal.last_update" has to be type "date" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:37.059	info	(17706) State value to set for "coronavirus-statistics.0.Germany.Kreis.Stendal.BL" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:36.819	info	(17706) State value to set for "coronavirus-statistics.0.Germany.Stadt.Magdeburg.last_update" has to be type "date" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:36.803	info	(17706) State value to set for "coronavirus-statistics.0.Germany.Stadt.Magdeburg.BL" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:32.537	info	(17706) State value to set for "coronavirus-statistics.0.Germany.Stadt.Emden.last_update" has to be type "date" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:32.520	info	(17706) State value to set for "coronavirus-statistics.0.Germany.Stadt.Emden.BL" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:31.756	info	(17706) State value to set for "coronavirus-statistics.0.Germany._Impfungen.rkiZweitimpfungenDifferenzVortag" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:31.740	info	(17706) State value to set for "coronavirus-statistics.0.Germany._Impfungen.rkiZweitimpfungenAstraZeneca" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:31.722	info	(17706) State value to set for "coronavirus-statistics.0.Germany._Impfungen.rkiZweitimpfungenModerna" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:31.673	info	(17706) State value to set for "coronavirus-statistics.0.Germany._Impfungen.rkiErstimpfungenDifferenzVortag" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:31.362	info	(17706) State value to set for "coronavirus-statistics.0.Germany.Bundesland.Sachsen-Anhalt._Impfungen.rkiZweitimpfungenDifferenzVortag" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:31.347	info	(17706) State value to set for "coronavirus-statistics.0.Germany.Bundesland.Sachsen-Anhalt._Impfungen.rkiZweitimpfungenAstraZeneca" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:31.328	info	(17706) State value to set for "coronavirus-statistics.0.Germany.Bundesland.Sachsen-Anhalt._Impfungen.rkiZweitimpfungenModerna" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:31.278	info	(17706) State value to set for "coronavirus-statistics.0.Germany.Bundesland.Sachsen-Anhalt._Impfungen.rkiErstimpfungenDifferenzVortag" has to be type "number" but received type "string"
coronavirus-statistics.0	2021-05-10 16:55:26.613	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.5.criticalPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.595	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.5.recoveredPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.578	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.5.activePerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.544	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.5.oneTestPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.526	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.5.oneDeathPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.507	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.5.oneCasePerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.472	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.5.population" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.449	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.5.testsPerOneMillion" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.427	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.5.tests" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.256	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.5.todayRecovered" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.117	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.4.criticalPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.096	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.4.recoveredPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.070	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.4.activePerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:26.012	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.4.oneTestPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.996	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.4.oneDeathPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.981	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.4.oneCasePerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.955	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.4.population" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.938	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.4.testsPerOneMillion" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.921	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.4.tests" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.841	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.4.todayRecovered" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.713	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.3.criticalPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.698	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.3.recoveredPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.684	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.3.activePerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.626	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.3.oneTestPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.611	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.3.oneDeathPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.597	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.3.oneCasePerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.572	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.3.population" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.558	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.3.testsPerOneMillion" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.544	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.3.tests" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.483	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.3.todayRecovered" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.352	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.2.criticalPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.333	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.2.recoveredPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.314	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.2.activePerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.283	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.2.oneTestPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.267	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.2.oneDeathPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.160	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.2.oneCasePerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.130	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.2.population" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.110	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.2.testsPerOneMillion" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.093	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.2.tests" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:25.024	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.2.todayRecovered" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:24.904	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.1.criticalPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:24.887	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.1.recoveredPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:24.872	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.1.activePerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:24.845	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.1.oneTestPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:24.829	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.1.oneDeathPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:24.813	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.1.oneCasePerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:24.784	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.1.population" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:24.768	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.1.testsPerOneMillion" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:24.752	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.1.tests" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:24.620	info	(17706) State value to set for "coronavirus-statistics.0.country_Top_5.1.todayRecovered" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:12.379	info	(17706) State value to set for "coronavirus-statistics.0.Germany.criticalPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:12.362	info	(17706) State value to set for "coronavirus-statistics.0.Germany.recoveredPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:12.344	info	(17706) State value to set for "coronavirus-statistics.0.Germany.activePerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:12.311	info	(17706) State value to set for "coronavirus-statistics.0.Germany.oneTestPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:12.291	info	(17706) State value to set for "coronavirus-statistics.0.Germany.oneDeathPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:12.273	info	(17706) State value to set for "coronavirus-statistics.0.Germany.oneCasePerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:12.236	info	(17706) State value to set for "coronavirus-statistics.0.Germany.population" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:12.216	info	(17706) State value to set for "coronavirus-statistics.0.Germany.testsPerOneMillion" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:12.187	info	(17706) State value to set for "coronavirus-statistics.0.Germany.tests" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:12.075	info	(17706) State value to set for "coronavirus-statistics.0.Germany.todayRecovered" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:10.845	info	(17706) State value to set for "coronavirus-statistics.0.global_totals.criticalPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:10.822	info	(17706) State value to set for "coronavirus-statistics.0.global_totals.recoveredPerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:10.795	info	(17706) State value to set for "coronavirus-statistics.0.global_totals.activePerOneMillion" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:10.754	warn	(17706) State attribute definition missing for + undefined
coronavirus-statistics.0	2021-05-10 16:55:10.752	info	(17706) State value to set for "coronavirus-statistics.0.global_totals.oneTestPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:10.677	info	(17706) State value to set for "coronavirus-statistics.0.global_totals.oneDeathPerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:10.612	info	(17706) State value to set for "coronavirus-statistics.0.global_totals.oneCasePerPeople" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:10.581	info	(17706) State value to set for "coronavirus-statistics.0.global_totals.population" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:10.532	info	(17706) State value to set for "coronavirus-statistics.0.global_totals.testsPerOneMillion" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:10.514	info	(17706) State value to set for "coronavirus-statistics.0.global_totals.tests" has to be type "string" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:10.398	info	(17706) State value to set for "coronavirus-statistics.0.global_totals.todayRecovered" has to be type "value" but received type "number"
coronavirus-statistics.0	2021-05-10 16:55:05.140	info	(17706) starting. Version 0.7.0-4 in /opt/iobroker/node_modules/iobroker.coronavirus-statistics, node: v12.22.1, js-controller: 3.3.8

```

**Comment by DutchmanNL** [comment]
solved release v0.7.0-5
